BAZProjectC1/Components/CarouselMovies/CarouselMovies.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  CarouselMovies.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 21/09/22.
+//
+
+import UIKit
+
+fileprivate let START_ROW: Int = 0
+fileprivate let START_SECTION: Int = 0
```

Comment:
Lo mismo con estas constantes, tratemos de usar lowerCammelCase
```suggestion
fileprivate let startRow: Int = 0
fileprivate let startSection: Int = 0
```